docs/source/quickstart/model_config.md

```diff
@@ -27,4 +35,27 @@ layers:
     name: final_dense
     units: 1
     activation: null
-```
\ No newline at end of file
+optimizer: 
+  key: adam
+  gradient_clip_value: 5.0       #default value is 5.0
+lr_schedule:
+  key: cyclic
+  type: triangular2
+  initial_learning_rate: 0.001   #default value is 0.001
+  maximal_learning_rate: 0.01    #default value is 0.01
+  step_size: 10                  #default value is 10
+```
+
+#Below is an example of defining other learning rate schedules.
+#lr_schedule:
+#  key: cyclic
+#  type: triangular
+#  initial_learning_rate: 0.001   #default value is 0.001
+#  maximal_learning_rate: 0.01    #default value is 0.01
+#  step_size: 10                  #default value is 10
+#
+#lr_schedule:
+#  key: exponential
+#  learning_rate: 0.01                   #default value is 0.01
+#  learning_rate_decay_steps: 100000   #default value is 100000
+#  learning_rate_decay: 0.96              #default value is 0.96
```

Comment:
```suggestion
Examples for defining other learning rate schedules:


Cyclic Learning Rate Schedule
```
#lr_schedule:
  key: cyclic
  type: triangular
  initial_learning_rate: 0.001   #default value is 0.001
  maximal_learning_rate: 0.01    #default value is 0.01
  step_size: 10                  #default value is 10
```

Exponential Decayed Learning Rate Schedule
```
lr_schedule:
  key: exponential
  learning_rate: 0.01                   #default value is 0.01
  learning_rate_decay_steps: 100000   #default value is 100000
  learning_rate_decay: 0.96              #default value is 0.96
```
```